create_marc_record.py

```diff
@@ -0,0 +1,136 @@
+from pymarc import Record, Field, Subfield
+
+
+def create_base_record() -> Record:
+    """Create a base MARC record, to which metadata from an external
+    source will be added."""
+    record = Record()
+
+    # Leader fixed field.
+    # Type (Leader/06) - j (Musical sound recording)
+    record.leader.type_of_record = "j"
+    # BLvl (Leader/07) - m (Monographic/Item)
+    record.leader.bibliographic_level = "m"
+    # Ctrl (Leader/08) - # (No specified type)
+    record.leader.type_of_control = "#"
+    # ELvl (Leader/17) - 3 (Abbreviated level)
+    record.leader.encoding_level = "3"
+    # Desc (Leader/18) - i (ISBD punctuation included)
+    record.leader.cataloging_form = "i"
+
+    # 007 - physical description fixed field
+    # 007 ## $a s $b d $d f $e u $f n $g g $h n $i n $m e $n u
+    # translated to fixed field:
+    record.add_field(Field(tag="007", data="sd#fungnn###eu"))
```

Comment:
In this 007, the $c (technically 007/02) should be blank, not `#`.

007/09 thru 007/11 should be fill characters `|`, not `#` (and not blank... MARC is so much fun!). Hermine's example has `007  sd\fungnn|||eu`, where 007/02 and other blanks are represented by `\`, which makes things even more confusing.

See https://www.loc.gov/marc/bibliographic/bd007s.html for details on the 007 for sound recordings.